### DISCUSSION TITLE
[Help] Is there any way to get value without calling renderAll

### DISCUSSION THREAD

**Discussion by faizulhaque** [fabricjs/fabric.js#9173] [Q&A] [answered]
I am changing to fontSize in for loop, but it's not updating the width of Text

Reference: 
https://jsfiddle.net/faizulhaque124/embdy45g/2/

However, if move the `canvas.add(text1);` before for loop, it works.

So just wondering, if is there any way to create the Text object and change the fontSize to get the width without calling renderAll all the time? (I believe renderAll function is expensive)

**Comment by ShaMan123** [comment, marked as answer]
call initDimensions
